### PR TITLE
Add Notify to collections publisher

### PR DESCRIPTION
**Pull request by 1pretz1**
Adds Notify to Collections Publisher and adds an action to send an email when content has been published without 2i review so content teams can keep track of who is publishing without review as this should only happen in exceptional circumstances.

Depends on the below being merged:
- [x] [govuk-secrets](alphagov/govuk-secrets#1000)
- [x] [govuk-puppet](alphagov/govuk-puppet#10457)

### Email sent in integration (before content review)
Screenshot 2020-06-25 at 13 21 36

Trello:
https://trello.com/c/NrZVDIlG/2018-3-alert-users-when-a-step-by-step-is-force-published-in-collections-publisher

### CODE REVIEW

File: app/mailers/publisher_notifications.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class PublisherNotifications < Mail::Notify::Mailer
+  include ApplicationHelper
+
+  TEMPLATE_ID = ENV.fetch("GOVUK_NOTIFY_TEMPLATE_ID", "test-template-id").freeze
```
Comment (benjamineskola) [inline app/mailers/publisher_notifications.rb:4]: We've implemented this differently in other places that use Notify (as a method on the base class) but I'm not sure it's important enough to change.

File: app/mailers/publisher_notifications.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class PublisherNotifications < Mail::Notify::Mailer
```
Comment (benjamineskola) [inline app/mailers/publisher_notifications.rb:1]: I think the convention is that `PublisherNotifications` (and all mailers) should inherit from `ApplicationMailer`, and then `ApplicationMailer` can be changed to inherit from `Mail::Notify::Mailer` — but again, not sure if this is important enough to warrant changing.

**Comment by 1pretz1** [comment]
I've updated the code to use an ApplicationMailer - I didn't do this at first as it seemed a bit overkill for one mailer action but consistency is king so it has been updated @benjamineskola!

**Comment by mattclear** [comment]
@1pretz1 For this one, I'd suggest:

>Hello,
>
>Peter Hartshorn has published a step by step without 2i review: 'Peters test – publish without 2i review'.
>
>You can view it here:
>https://www.integration.publishing.service.gov.uk/peters-test-test
>
>Sent by the GOV.​UK Collections Publisher application

**Pull request merged**